services/app-web/src/pages/Settings/EditStateAssign/EditStateAssign.test.tsx

```diff
@@ -146,22 +149,25 @@ describe('EditStateAssign', () => {
 
     it('cancel button moves admin user back to state assignments table', async () => {
         let testLocation: Location // set up location to track URL change
-        const { user } = renderWithProviders(wrapInRoutes(<EditStateAssign />), {
```

Comment:
are there any changes in these app-api/app-web files that aren't whitespace?

Reply:
@JasonLin0991 saw something like this on my PR too. I think precommit isn't working anymore  -- both linting and prettier don't seem to be totally synced. I keep fixing others lint errors. And these are my prettier errors (makes sense I dont have prettier in my vscode right now I recently had to reset all my extensions and forgot to add it). But thats the point of precomit to smooth this out before merge.